emloop/cli/common.py

```diff
@@ -91,10 +91,10 @@ def create_dataset(config: dict, output_dir: Optional[str]=None) -> AbstractData
     return dataset
 
 
-def create_model(config: dict, output_dir: Optional[str], dataset: AbstractDataset,
+def create_model(config: dict, output_dir: Optional[str]=None, dataset: Optional[AbstractDataset]=None,
```

Comment:
I would love having all the `create_*` methods accepting both config filename (str) and config dict (otherwise/dict).

Reply:
I don't agree with this. I think there should be one way how to do something with an API not two ways. And part of the API is function `load_yaml` which allows you to convert the filename to dict.